en/micro-integrator/docs/setup/deployment/kubernetes_deployment.md

```diff
@@ -1,4 +1,4 @@
-# Deploy on Kubernetes
+# Deploy on Kubernetes using EI-K8s Operator
```

Comment:
shall we change this as K8s EI-Operator?

Reply:
Yes.. you are right.. 
The git repo is K8s-EI Operator: https://github.com/wso2/k8s-ei-operator
I think we will have to update all instances of 'EI-K8s Operator'.

Reply:
+1. Fixed